js/summary_test.go

```diff
@@ -23,9 +23,9 @@ const (
 		"       ✓ check1\n" +
 		"       ✗ check3\n        ↳  66% — ✓ 10 / ✗ 5\n" +
 		"       ✗ check2\n        ↳  33% — ✓ 5 / ✗ 10\n\n" +
-		"   ✓ checks......: 75.00% ✓ 45  ✗ 15 \n"
+		"   ✓ checks......: 75.00% 45 out of 60\n"
```

Comment:
It seems inconsistent with the `check1`, `check2`, and `check3` since they still use icons. Do you think so?

Reply:
I think the reason is because one is stored as a `Rate` metric, so it is handled as such, while the other falls under the `summarizeCheck` path.

That said, as I mentioned in the description, I didn't modify `summarizeCheck` because I feel that the use of icons for that may still be useful.

However, it's true that once all together, it looks a bit inconsistent.
So, would you suggest to use `out of` everywhere?
cc/ @oleiade @codebien @mstoykov  (as you already shared some feedback before around this discussion) 🙇🏻

Reply:
Isn't still dependent from what is the meaning of the check? For example if `check3` would be something like `whatever_metric_failed` wouldn't be still confusing?

Reply:
Well... yes and no 😅

Technically, a check that succeeds is always ✓, while a rate metric is just a number.
Think analogously in programming terms, if the code execution enters into an `if` statement, it's always because the condition is satisfied, even if the condition reflects a negative event (e.g. `if failed { ... }`), but a rate is just a variable that holds a number, where the number might reflect succeeds or fails. So, the ambiguity is different/smaller.

That said, I agree that could make sense to change it everywhere. So, let's open the discussion widely and see what other people prefers, honestly I don't have ~strong~ preferences.